workday/13.0/modules/ROOT/pages/workday-connector-design-center.adoc

```diff
@@ -1,66 +1,57 @@
 the xref:workday-reference.adoc[Workday Connector Reference].
 
-== Configure the Input Source Trigger
+== Configure the Trigger
 
-A trigger starts your app. Workday Connector provides the Modified Object Trigger operation that starts your app when an object is created, modified, or deleted in Workday. Alternatively, you can use the HTTP Listener operation to start your app from a browser
-or with a command such as `curl`. Another alternative is to use Scheduler to start your app on a timed schedule.
+A trigger starts your app. Workday Connector provides the *On Modified Objects* trigger that starts your app when an object is created, modified, or deleted in Workday. Alternatively, you can use:
 
-. In Design Center, click *Create*.
+* *HTTP Listener* to start your app from a browser
+or with a command such as `curl`.
+* *Scheduler* to start your app on a timed schedule
+
+To configure the trigger:
+
+. In Design Center, click *Create new*.
 . Click *Create new application*.
 . Specify a value for *Project name*, and click *Create*.
-. Click *Go straight to canvas* to exit from *Let's get started* and go to Flow Designer.
-. Click the trigger card.
-. If you are using Workday Connector as the input source trigger, search for the connector;
-otherwise, you can use HTTP or Scheduler.
-. Complete these fields when using the Modified Object Trigger operation:
+. Click *Go straight to canvas* to exit from *Let's get started*.
+. Click the empty trigger card in the middle of the canvas to expand it.
+. Configure a trigger for the connector.
 +
-[%header,cols="30s,55a,15a"]
-|===
-|Field |Description |Required?
-|Configuration |Name of the configuration to use. |Yes
-|Object Types |Object type that triggers the poll. |Yes
-|Scheduling Strategy |Scheduler configuration that triggers your app by polling. |Yes
-|Frequency |Polling frequency in time units. |No
-|Start delay|Time that Scheduler waits before starting. |No
-|Time Unit|Time unit for *Frequency* and *Start delay* values. |Yes
-|===
-+
-. Click *Test*.
-
-Design Center automatically saves all changes.
+To use the *On Modified Objects* trigger, enter `workday` in the search field, select *Workday Connector*.
+. Click *Add Connection*.
+. In the *Connection Name field*, enter a name for the connection.
+. In the *Connection Type* field, select whether to use Basic authentication or x.509 Authentication.
+. Fill in the required fields.
+. Click *Test* to test the connection.
+. Click *Save*.
 
 == Configure the Target Component
 
-The target component enables your app to process data received from the trigger. Workday Connector
-provides operations that you can use to perform tasks for educational organizations, human resources, professional services,
+The target component enables your app to process data received from the trigger. Workday Connector provides operations that you can use to perform tasks for educational organizations, human resources, professional services,
```

Comment:
"target component" here is a type of component, not the name (Target component, per style ref) of a MuleSoft component, right? Just making super sure....